kubectl/instrumentation/elasticsearch/elasticsearch-data-statefulset.yaml

```diff
@@ -49,6 +49,15 @@ spec:
                 topologyKey: failure-domain.beta.kubernetes.io/zone
       securityContext:
         fsGroup: 1000
+      initContainers:
+      # see https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
+      # and https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration-memory.html#mlockall
+      - name: "sysctl"
+        image: "busybox"
+        imagePullPolicy: "Always"
+        command: ["sysctl", "-w", "vm.max_map_count=262144"]
+        securityContext:
+          privileged: true
```

Comment:
@cherwin mentioned this in a thread https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/

Have you looked at that? Running privileged containers is best avoided

Reply:
Thanks - re-working this now to add an annotation to the pods, and a value to the kubelet to allow sysctl changes